docs/developer_guide/tpu_resources_guide.md

```diff
@@ -0,0 +1,108 @@
+# Guide
+
+## Skypilot Installation/Login
+
+```bash
+# Refer to https://docs.skypilot.co/en/latest/getting-started/installation.html
+pip install skypilot
+
+sky api login
+```
+
+Enter your SkyPilot API server endpoint: Please join the [SGL-JAX Slack](https://sgl-fru7574.slack.com/archives/C09EBE5HT5X) and gain it from maintainers.
+
+## Generate a Yaml
+
+```yaml
+Example Yaml
+resources:
+   accelerators: tpu-v6e-4 #
+   accelerator_args:
+      tpu_vm: True
+      runtime_version: v2-alpha-tpuv6e  # optional
+file_mounts:
+  ~/.ssh/id_rsa: ~/.ssh/id_rsa
+setup: |
+  chmod 600 ~/.ssh/id_rsa
+  rm ~/.ssh/config
+  GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" git clone git@github.com:sgl-project/sglang-jax.git
+```
```

Comment:
Perhaps we should remind you that if you are testing small models (models smaller than 100B) it is best to mount tmpfs to /tmp to prevent running out of disk space.
`sudo mount -t tmpfs -o size=200G tmpfs /tmp`